src/containers/App/themeConstants.js

```diff
@@ -37,7 +37,7 @@ export const TEMPLATE_LIBRARY = {
   HEADER_TITLE: '#939EBA',
   ACTION_BUTTON: AP_THEME.CYAN,
   ACTION_BUTTON_BACKGROUND: AP_THEME.DARK_BLUE_MEDIUM,
-  ACTION_BUTTON_BORDER: '#50637F',
+  ACTION_BUTTON_BORDER: '#7B8FAD',
```

Comment:
I think this is the border, not the text.

Reply:
But its affecting the color

Reply:
AHH! So sorry, you're right.